ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Pre-defined styles of icons
+@objc(MSFListIconVnextStyle)
+public enum MSFListIconVnextStyle: Int, CaseIterable {
+    case none
+    case iconOnly
+    case large
+}
+
+/// Pre-defined types of icons
+@objc(MSFListAccessoryType)
+public enum MSFListAccessoryType: Int, CaseIterable {
+    case none
+    case disclosure
+    case detailButton
+    case checkmark
+
+    var icon: UIImage? {
+        let icon: UIImage?
+        switch self {
+        case .none:
+            icon = nil
+        case .disclosure:
+            icon = UIImage.staticImageNamed("iOS-chevron-right-20x20")
+        case .detailButton:
+            icon = UIImage.staticImageNamed("more-24x24")
+        case .checkmark:
+            icon = UIImage.staticImageNamed("checkmark-24x24")
+        }
+        return icon
+    }
+}
+
+///Properties that make up cell content
+@objc(MSFListVnextCellData)
+public class MSFListVnextCellData: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingIcon: UIImage?
+    @objc @Published public var title: String = ""
+    @objc @Published public var subtitle: String?
+    @objc @Published public var trailingIcon: MSFListAccessoryType = .none
+    @objc @Published public var titleLineLimit: Int = 1
+    @objc @Published public var subtitleLineLimit: Int = 1
+    @objc public var onTapAction: (() -> Void)?
+}
+
+///Properties that make up section content
+@objc(MSFListVnextSectionData)
+public class MSFListVnextSectionData: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var cells: [MSFListVnextCellData] = []
+    @objc @Published public var title: String?
+    @objc @Published public var layoutType: MSFListCellVnextLayoutType = .oneLine
```

Comment:
Wouldn't it be nicer to allow the calling code to specify the layout type per cell (in the MSFListVnextCellData)?
We can discuss this after your first merge (it's worth a GitHub issue to track as well).

Reply:
I wanted to consider both cell design + consistency... should we allow sections to contain cells of different heights? I agree we should continue this in a discussion, I will create an issue

Reply:
https://github.com/microsoft/fluentui-apple/issues/367